content/docs/3_reference/3_panel/3_fields/0_color/reference-article.txt

```diff
@@ -94,7 +94,7 @@ color:
 <since v="4.1.0">
 ### Dynamic options
 
-Our options (link: docs/guide/blueprints/query-language text: query syntax) offers a very powerful way of to add an automatically generated option lists to a color field.
+Our options (link: docs/guide/blueprints/query-language text: query syntax) offer a powerful way of adding automatically-generated option lists to a color field.
```

Comment:
Is this right? `syntax` is singular and so shouldn't it be `offers`? `automatically generated` should also not have a hyphen, it's not a composable, is it?

Reply:
Correct about 'offer/offers', misread it in code mode.